### PR TITLE
Requirements table now shows auto-assign attribute

### PR DESCRIPTION
**Pull request by DrKalen**
I'm not sure how to create a test for this in RSpec. Should I create a new Issue for that?

### CODE REVIEW

File: app/views/requirements/index.html.erb [hunk 1]
```diff
@@ -29,6 +29,7 @@
         <td><%= requirement.minimum_people %></td>
         <td><%= requirement.maximum_people %></td>
         <td><%= requirement.optional ? "Yes" : "No" %></td>
+        <td><%= requirement.auto-assign ? "Yes" : "No" %></td>
```
Comment (kgf) [inline app/views/requirements/index.html.erb]: I don't think this is the correct name for the attribute. Underscore, not hyphen.
https://github.com/ReadyResponder/ReadyResponder/blob/development/db/schema.rb#L437

Comment (kgf) [reply app/views/requirements/index.html.erb]: Did you try this on your development instance ?

Comment (DrKalen) [reply app/views/requirements/index.html.erb]: Right you are! I've made the correction and will put in a new pull request.

I don't know how to set up a development instance to try it on. Where can I find out how to do that?

Comment (DrKalen) [reply app/views/requirements/index.html.erb]: I guess the change to auto_assign is now the most recent version on this pull request? (Still learning how that works!)

**Comment by kgf** [comment]
Sorry I missed you at project night. 
To set up a development environment, you can follow the directions here:
https://github.com/ReadyResponder/ReadyResponder#getting-started---dependencies

**Comment by DrKalen** [comment]
Thanks Kevin!
Best,
Kalen


Kalen Hammann
Software Engineer
Cell: 303-589-7107 <1-303-589-7107>
Portfolio/Apps:
     http://kalenhamman <http://kalenhammann.getforge.io/>.com
     https://github.com/DrKalen
     https://www.linkedin.com/in/kalenhammann/

**Comment by kgf** [comment]
@DrKalen Are you having any success ? Need any additional help ?

**Comment by DrKalen** [comment]
I redid the commit from ground zero.

**Comment by houhoulis** [comment]
This isn't ready for merge because the Auto-Assign header column isn't lined up with each requirement's auto-assign field.  The header is just after `<th>Maximum</th>`, so I imagine the auto-assign fields should likewise be just after `<td><%= requirement.maximum_people %></td>`.

(We were rushing, sorry)

**Comment by houhoulis** [comment]
@DrKalen , we also need to make sure that we are fixing the problem that was [identified in the issue](https://github.com/ReadyResponder/ReadyResponder/issues/600).  It's good to update the Requirements index page too, but the Issue states that the _`tasks#show` page_ needs to show the auto-assign field.  Let me know if you need help identifying how to fix that page.

**Comment by DrKalen** [comment]
Hi Chris -
Thanks for your help last night, and for the follow-up.
I thought I HAD put the header in the same order as the field (after
optional). Guess I was moving too fast! I'll change that.

And it's embarrassing that I fixed the wrong page.  As to fixing the
tasks#show page, I see that it has a tab rendering part of  a
requirements/task_requirements_table. I'm guessing  that means the file
named requirements/_task_requirements_table needs to be changed as well,
and I can see that it's missing the auto_assign header and field. But I
don't know how to know if that is all that's needed to fix the tasks#show
page.

Best,
Kalen


Kalen Hammann
Software Engineer
Cell: 303-589-7107 <1-303-589-7107>
Portfolio/Apps:
     http://kalenhamman <http://kalenhammann.getforge.io/>.com
     https://github.com/DrKalen
     https://www.linkedin.com/in/kalenhammann/

**Comment by houhoulis** [comment]
@DrKalen , I believe you've identified the right partial to fix -- the "_task_requirements_table" file.

> _But I don't know how to know if that is all that's needed_

The advantage of running `bundle exec rails server` locally is you can see whether your changes have the effect you want!

**Comment by DrKalen** [comment]
Fair enough! I'll do that.

Thanks,
Kalen


Kalen Hammann
Software Engineer
Cell: 303-589-7107 <1-303-589-7107>
Portfolio/Apps:
     http://kalenhamman <http://kalenhammann.getforge.io/>.com
     https://github.com/DrKalen
     https://www.linkedin.com/in/kalenhammann/

**Comment by kgf** [comment]
@DrKalen Are you still working on this ? Should I close this PR, and you can open another when you're ready ?
This is in regard to issue #600.

**Comment by kgf** [comment]
I did not hear anything back. This can be re-opened at any time.

**Comment by DrKalen** [comment]
Hi Kevin -
So sorry. I’ve been otherwise occupied (a learning blitz for a new job) and missed your email. Glad you went ahead and closed it. 
I hope I’ll be able to get back to Ready Responder at some point in the future. 

Best regards, Kalen

**Pull request closed without merging**